src/moonlink/src/storage/mooncake_table.rs

```diff
@@ -966,6 +1010,24 @@ impl MooncakeTable {
         );
     }
 
+    /// Update table schema to the provided [`updated_table_metadata`].
+    /// Notice: schema evolution performs IO operation in blocking style, and only returns when completion.
+    #[allow(dead_code)]
+    pub(crate) async fn alter_table_schema(
```

Comment:
@zhousun I assume this is the function you will integrate into table handler, iceberg schema evolution is implemented inside.